pkg/api/v1beta3/dynakube/oneagent_props.go

```diff
@@ -228,6 +228,35 @@ func (dk *DynaKube) DefaultOneAgentImage(version string) string {
 	return apiUrlHost + DefaultOneAgentImageRegistrySubPath + ":" + tag
 }
 
+func (dk *DynaKube) OneAgentArguments() map[string][]string {
```

Comment:
I would expect this to just return the `[]string` as the `Args` is a `[]string` 

Trying to come up with a better name, but failing 😅 
(I like what it's doing, I just don't like the name)

Reply:
The idea is to have a single location where the arguments are properly "parsed". And the function returns the OneAgent arguments. It's about what is returned not in which format.

Reply:
> the arguments are properly "parsed"

that is kinda the problem, other function that is just basically name of the field you are getting does no parsing (OneAgentNodeSelector, OneAgentSecCompProfile, OneAgentNamespaceSelector)
- So my main problem is that you are going against the current pattern

on `main` this would read `dk.OneAgent().Arguments()`, and someone might use it to get the that inner field `dk.Spec.OneAgent.CloudNativeFullStack.Args`, which is a `[]string` but they will weirdly receive a `map[string]string`
- you could call it `dk.OneAgentArgumentsMap()` which is a mouthful, but considering that in `main`it would be `dk.OneAgent().ArgumentsMap()` which is IMO not that bad